signalrclient_VS2013.sln

```diff
@@ -18,48 +18,116 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{063421
 		.nuget\NuGet.targets = .nuget\NuGet.targets
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "sample", "sample", "{046E5264-9A01-4047-9FF4-02B76B30C727}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "signalr-server-selfhost", "sample\signalr-server-selfhost\signalr-server-selfhost.csproj", "{11848039-1F13-4047-9539-8F9F45930788}"

```

Comment:
The host for testing should be in test and not in sample. This project is C++ Client so we don't need to show all different things you can do with the server (especially that we do some different hacks like - throwing for reconnects which does not make a good sample). The samples will show how to use clients. I believe we have comprehensive samples in the SignalR repo for the server and we don't need to copy them here and maintain two copies.